Field.php

```diff
@@ -30,18 +30,60 @@ public function __construct($_width, $_height, $_maxSteps)
         $this->width = $_width;
         $this->height = $_height;
         $this->maxSteps = $_maxSteps;
-        for ($y = 0; $y < $this->height; $y++) {
-            for ($x = 0; $x < $this->width; $x++) {
+
+        $this->constructField();
+    }
+
+    /**
+     * Construct empty field
+     */
+    public function constructField()
```

Comment:
Is there a reason that this method is public? It could (and probably should) be private, or not?